src/Kernel/Helpers/GetEnvironmentVariableHelper.cs

```diff
@@ -0,0 +1,14 @@
+﻿using LT.DigitalOffice.Kernel.Exceptions.Models;
+using System;
+
+namespace LT.DigitalOffice.Kernel.Helpers
+{
+    public static class GetEnvironmentVariableHelper
+    {
+        public static string Get(string key)
+        {
+            return Environment.GetEnvironmentVariable(key)
+                ?? throw new NotFoundException($"No variable named {key}");
```

Comment:
not sure that exception right

Reply:
What do you want? What type of exception?